projects/etcd-io/etcd/1-21/GIT_TAG

```diff
@@ -1 +1 @@
-v3.4.21
+v3.4.20
```

Comment:
Let's not downgrade, in this case I know we wanted to upgrade to solve a bug in etcd, but even if we did it by accident I don't think we ever want to downgrade.